help/de/Help_ch_Account-Actions.xml

```diff
@@ -90,336 +104,317 @@
-                eine gleitende Kreditlinie erlauben (z. B. VISA, oder MasterCard) als auch bei
-                Karten, die keine laufenden Guthaben zulassen (z. B. American Express), verwendet
-                werden.
+              <para>Aktienkonto
+              </para>
+            </entry>
+
+            <entry namest="c3">
+              <para>Die Aktie (englisch share) ist ein Wertpapier, das den Anteil an einer Aktiengesellschaft oder einer
+                Kommanditgesellschaft auf Aktien (KGaA) verbrieft.
+                <footnote>
+                  <para><ulink url="&url-wp-de;Aktie"/>
+                  </para>
+                </footnote>
+
+                <footnote>
+                  <para><tip>
+                      <para>Solange Crypto-Währungen von &app; nicht als Währungen unterstützt, wird empfohlen, sie auch als
+                        Aktie anzulegen.
+                      </para>
+                    </tip>
+                  </para>
+                </footnote>
               </para>
             </entry>
           </row>
 
           <row>
-            <entry namest="c1">
-              <para>Devisenhandel
+            <entry namest="c2" rowsep="1">
+              <para>Investmentfonds
               </para>
             </entry>
 
-            <entry namest="c2">
-              <para>Währungskonten wurden für den Handel mit Devisen verwendet, sind aber durch den Bereich
-                "Währungstransfer" im Fenster "Geldtransfer" ersetzt worden
-                (<xref linkend="trans-win-enter"></xref>).
+            <entry namest="c3" rowsep="1">
+              <para>Ein Portfolio aus Aktien, Anleihen oder anderen Geldanlagen, das sich in Anteile teilen läßt.
               </para>
             </entry>
           </row>
 
           <row>
-            <entry namest="c1">
-              <para>Eigenkapital
-              </para>
-            </entry>
-<!-- FIXME source: insert link and <menuchoice> -->
-            <entry namest="c2">
-              <para>Eigenkapitalkonten werden dazu verwendet, die Eröffnungsbestände beim ersten Start von &app; sowie
-                das Ergebnis eines <xref linkend="tool-close-book" /> (
-                <menuchoice>
-                  <guimenu><accel>W</accel>erkzeug</guimenu>
-                  <guimenuitem>Buch<accel>a</accel>bschluss</guimenuitem>
-                </menuchoice>
-                ) am Ende einer Buchungsperiode zu erfassen.
+            <entry namest="c1" morerows="2" valign="top">
+              <para>Passiva
               </para>
             </entry>
 
             <entry namest="c3">
-              <para>nur Eigenkapitalkonten
+              <para>Passiva, in &app; als Fremdkapital genannt, bezeichnet Schulden oder Verbindlichkeiten und stellen
+                das auf der rechten Bilanzseite verzeichnete Eigen- und Fremdkapital eines
+                Unternehmens dar.
```

Comment:
Das gehört oben in Passiva, sofern nicht redundant.

Reply:
ok.